Add airodump content in devops roadmap (#2634)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/117-airodump.md

> # Airodump
> 
> It's a packet sniffer; used for packet captures such as WEP IVs (Initialization Vector) and WAP Handshake. Even if you have a GPS receiver connected to the computer, you can log the coordinates of the found access points.
> 
> This tool is a part of a network software suite called Aircrack-ng that includes:
> * [airbase-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airbase-ng)
> * [aircrack-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=aircrack-ng)
> * [airdecap-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airdecap-ng)
> * [airdecloak-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airdecloak-ng)
> * [airdrop-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airdrop-ng)
> * [aireplay-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=aireplay-ng)
> * [airgraph-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airgraph-ng)
> * [airmon-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airmon-ng)
> * [airodump-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airodump-ng)
> * [airolib-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airolib-ng)
> * [airserv-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airserv-ng)
> * [airtun-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=airtun-ng)
> * [besside-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=besside-ng)
> * [dcrack](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=dcrack)
> * [easside-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=easside-ng)
> * [packetforge-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=packetforge-ng)
> * [tkiptun-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=tkiptun-ng)
> * [wesside-ng](https://aircrack-ng.org/~]%C3%AF%C2%BF%C2%BD%C3%AF%C2%BF%C2%BD/doku.php?id=wesside-ng)
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink badgeText='Official Website' colorScheme='blue' href='https://www.aircrack-ng.org/'>Official Website</BadgeLink>
> <BadgeLink badgeText='Official Documentation' colorScheme='blue' href='https://www.aircrack-ng.org/doku.php'>Official Documentation</BadgeLink>
> <BadgeLink badgeText='Watch' colorScheme='yellow' href='https://www.youtube.com/watch?v=ftM-Fra90VM'> Airodump ng Tool and Capture WPA WPA2 Handshake </BadgeLink>
> <BadgeLink badgeText='Watch' colorScheme="red" href='https://www.youtube.com/watch?v=O6jYy421tGw'>  Airodump ng Tool and Capture WPA WPA2 Handshake (by MBP Security on YouTube)</BadgeLink>

Reviewer: ```suggestion
# Airodump

It's a packet sniffer; used for packet captures such as WEP IVs (Initialization Vector) and WAP Handshake. Even if you have a GPS receiver connected to the computer, you can log the coordinates of the found access points.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink badgeText='Official Website' colorScheme='blue' href='https://www.aircrack-ng.org/'>Official Website</BadgeLink>
<BadgeLink badgeText='Official Documentation' colorScheme='blue' href='https://www.aircrack-ng.org/doku.php'>Official Documentation</BadgeLink>
<BadgeLink badgeText='Watch' colorScheme='red' href='https://www.youtube.com/watch?v=ftM-Fra90VM'> Airodump ng Tool and Capture WPA WPA2 Handshake</BadgeLink>
<BadgeLink badgeText='Watch' colorScheme="red" href='https://www.youtube.com/watch?v=O6jYy421tGw'>  Airodump ng Tool and Capture WPA WPA2 Handshake (by MBP Security on YouTube)</BadgeLink>
```

# Airodump

> It's a packet sniffer; used for packet captures such as WEP IVs (Initialization Vector) and WAP Handshake. Even if you have a GPS receiver connected to the computer, you can log the coordinates of the found access points.

Reviewer: ```suggestion
It's a packet sniffer used for packet captures such as WEP IVs (Initialization Vector) and WAP Handshake. Even if you have a GPS receiver connected to the computer, you can log the coordinates of the found access points.
```
